PostBy/View Controllers/SettingsViewController.m

```diff
@@ -44,6 +55,41 @@ - (void)viewDidLoad {
     PFFileObject *profilePicObj = [PFUser.currentUser valueForKey:@"profilePicture"];;
     NSURL *url = [NSURL URLWithString:profilePicObj.url];
     [self.profilePicture setImageWithURL:url];
+    
+    self.INVALID_CHOICE = -1;
+    self.DELETE_CHOICE = self.INVALID_CHOICE;
+    [self createDropDown];
+}
+
+- (void)createDropDown {
+    CGFloat x = (CGRectGetWidth(self.view.frame)-240)/2;
```

Comment:
Trying to understand what "240" means in this line... 🤔
Should we give it a name?

Reply:
Fixed! It refers to the width below. I change the order of declarations and swapped `240` for `width`